src/localization/en-us/commands.json

```diff
@@ -346,6 +346,9 @@
     "suggestionRejectedColor": "0xcc0000",
     "suggestionRejectedTitle": "Suggestion Rejected.",
     "suggestionRejectedDescription": "A suggestion sent by {user-mention} was rejected.",
+    "suggestionImplementedColor": "1030B0",
```

Comment:
Not sure if Discord accepts hex without the `0x` prefix. Even if they work I would suggest changing them to fit the convention used by the other ones.